### ISSUE TITLE
Gramatical Error: "a inefficient"

### ISSUE THREAD

**Issue by jaredririe** [marijnh/Eloquent-JavaScript#310]
http://eloquentjavascript.net/3rd_edition/03_functions.html#p_t57qMezbeH

In the following paragraph under the heading **Recursion**, "a inefficient" should be written "an inefficient".
```
Recursion is not always just a inefficient alternative to looping. Some problems are really easier to solve with recursion than with loops. Most often these are problems that require exploring or processing several “branches”, each of which might branch out again into more branches.
```

**Comment by JBallin** [comment]
I'll submit a PR for this shortly

**Comment by marijnh** [comment]
Thanks for spotting that. Solved via #321
